doc/whats_new/v1.3.rst

```diff
@@ -34,6 +34,12 @@ random sampling procedures.
   are strings. Predefined categories encoded as bytes will no longer work
   with `X` encoded as strings. :pr:`25174` by :user:`Tim Head <betatim>`.
 
+- |Fix| NaN values are now ignored to estimate quantiles in
```

Comment:
This change log entry is slightly different from the one below. Is it on purpose that they are repeated? If yes, I prefer this version to the one below.